packages/cyberstorm/src/components/Tag.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactNode } from "react";
+import styles from "./componentStyles/Tag.module.css";
+
+export interface ButtonProps {
+  label?: string;
+  leftIcon?: ReactNode;
+  rightIcon?: ReactNode;
+  tagStyle?: string;
+}
+
+/**
+ * Cyberstorm Tag component
+ */
+export const Tag: React.FC<ButtonProps> = (props) => {
+  const { label, tagStyle, leftIcon, rightIcon } = props;
+  styles; //if styles is not called, the classes from the css module aren't found
```

Comment:
This is not required. See https://github.com/thunderstore-io/thunderstore-ui/pull/569. Rebase this PR on top of that one and use the CSS modules via the imported values rather than strings as className values.

Reply:
Done